examples/8760.jl

```diff
@@ -8,12 +8,14 @@ filename = joinpath(@__DIR__, "ckt5", "Master_ckt5.dss")
 
     Solve  ! This executes a snaphot power flow solution of the circuit
 
-    set mode=yearly
-    set demand=true
-    set DIVerbose=true
-    set voltexceptionreport = true
-    set Year=1        ! This statement resets all Meters
-    solve
-    closeDI
+    ! set mode=yearly
```

Comment:
Why are these commented out? The point of the file is to run an 8760-hour simulation.

Reply:
This was taking a really long time to run on my machine. I'll try again and see exactly how long it takes to finish.

Reply:
Fixed in cc7d20d.